src/software/simulated_tests/or_validation.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+import software.python_bindings as tbots
```

Comment:
nit: unused imports

This is a great validation tho, a lifesaver in my PR

Reply:
Thanks! And fixed.